docs/content/architecture/user-management.md

```diff
@@ -48,18 +48,37 @@ Users and databases can be customized in the `spec.users` section of the custom
 - For any users added in `spec.users`, PGO will created a Secret of the format `<clusterName>-pguser-<userName>`. This will contain the user credentials.
   - If no databases are specified, `dbname` and `uri` will not be present in the Secret.
   - If at least one `spec.users.databases` is specified, the first database in the list will be populated into the connection credentials.
-- To prevent accidental data loss, PGO will not automatically drop users. We will see how to drop a user below.
-- Similarly, to prevent accidental data loss PGO will not automatically drop databases. We will see how to drop a database below.
+- To prevent accidental data loss, PGO does not automatically drop users. We will see how to drop a user below.
+- Similarly, to prevent accidental data loss PGO does not automatically drop databases. We will see how to drop a database below.
 - Role attributes are not automatically dropped if you remove them. You will have to set the inverse attribute to drop them (e.g. `NOSUPERUSER`).
 - The special `postgres` user can be added as one of the custom users; however, the privileges of the users cannot be adjusted.
 
 For specific examples for how to manage users, please see the [user and database management]({{< relref "tutorial/user-management.md" >}}) section of the [tutorial]({{< relref "tutorial/_index.md" >}}).
 
+## Generated Passwords
+
+PGO generates a random password for each Postgres user it creates. Postgres allows almost any character
+in its passwords, but your application may have stricter requirements. When you need a password without
+any special characters set `spec.users.password.type` for that user to `AlphaNumeric`.
```

Comment:
⛏️ "characters set" sounds like "character set". 🤔 Is there supposed to be a comma here?